cli/installer/docker_compose.go

```diff
@@ -259,6 +260,8 @@ func fixTracetestContainer(config configuration, project *types.Project, version
 	tts.Image = "kubeshop/tracetest:" + version
 	tts.Build = nil
 	tts.Volumes[0].Source = tracetestConfigFilename
+	tracetestDevEnv := "${TRACETEST_DEV}"
+	tts.Environment["TRACETEST_DEV"] = &tracetestDevEnv
```

Comment:
We lose this value the moment we load the yaml config because it gets evaluated as an empty string.